public/help.md

```diff
@@ -58,36 +58,35 @@ Ta kontakt med en systemadministrator for å få brukernavn og passord til bruke
 
 ## KPI-er
 
-KPI-verdiene kan ikke oppdateres direkte i OKR-trackeren.
-
-KPI-en må knyttes til et Google Sheet-dokument med verdien som lastes automatisk inn i OKR-trackeren med gitt tidsintervall (to ganger daglig).
+### Legge til ny KPI
 
-Viktig: Google Sheets-dokumentet må **deles** med OKR-trackerens _service account_. Adressen til denne finner du under «Sheets-detaljer» når du oppretter ny KPI.
+#### Steg 1: Fyll inn detaljer
 
-### KPI-typer
+- Naviger til ditt produkt (eventuelt produktområde eller organisasjon for systemadministratorer).
+- Klikk «Endre \<navn\>».
+- Velg «KPI-er».
+- Klikk «Legg til KPI».
+- I skjemaet må du velge KPI-type (se over), navn og beskrivelse
+  (valgfritt). Her ser du også e-post-adressen (service account) som Google
+  Sheets-dokumentet må deles med.
 
-Det finnes tre ulike KPI-er:
+#### Steg 2 (valgfritt): Sette opp Google Sheets
 
-- _Volum_ (absolutt verdi)
-- _Brukertilfredshet_ (prosent)
-- _Fullføringsgrad_ (prosent)
+- Se [Google Sheets-integrasjon](#google-sheets-integrasjon).
 
-Disse KPI-ene kan legges til organisasjoner, produktområder og produkter.
+### Oppdatere KPI-verdier
 
-### Legge til ny KPI
+KPI-verdiene kan oppdateres på tre ulike måter:
 
-#### Steg 1: Sette opp Google Sheets
+1. Direkte i OKR-trackeren ved å benytte knappen «Legg til målepunkt».
 
-- Se [Google analytics-integrasjon](#google-analytics-integrasjon).
+2. Via API-et. Instruksjoner for dette finner du når du redigerer eller
+   oppretter en ny KPI.
 
-#### Steg 2: Fyll inn detaljer
-
-- Naviger til ditt produkt (eventuelt produktområde eller organisasjon for systemadministratorer).
-- Klikk «Endre \<navn\>».
-- Velg «KPI-er».
-- Klikk «Legg til KPI».
-- I skjemaet må du velge KPI-type (se over), navn og beskrivelse (valgfritt). Her ser du også e-post-adressen (service account) som Google Sheets-dokumentet må deles med. Google Sheet ID-en finner du i URL-en til dokumentet: `.../spreadsheets/d/<dokument-id>/edit`
-- Trykk «Legg til» og vent til du får grønn «OK» for bekreftelse på gyldig kobling.
+3. Ved å knytte KPI-en til et Google Sheets-dokument vil verdier fra dokumentet
+    lastes inn automatisk i OKR-trackeren to ganger daglig. Google
+    Sheets-dokumentet må deles med OKR-trackerens _service account_. Adressen
+    til denne finner du når du redigerer eller oppretter en ny KPI.
```

Comment:
Hvor?

Reply:
Hrrrm, ser ut som den ble fjerna i 1c592a5a3af1d0b0efc8c1894130fb96b36d6412. :thinking: Får legge den til igjen, da? For den adressen trenger man for å kunne gi tilgang.

Reply:
Hm, må kanskje det? Har egentlig ikke tenkt så mye på det, utover at jeg antok man benyttet deling med "Alle som har lenken" - og det er muligens ikke det beste. Får vi den inn i Google Sheets-seksjonen på en grei måte eventuelt? 🤔

Reply:
Skal prøve.